tests/distributed/test_partition.py

```diff
@@ -1,17 +1,21 @@
-import dgl
-import sys
 import os
```

Comment:
Let's avoid autoformat on existing code together with other code change, very hard to review :(

Reply:
The reason why I formatted here is I changed much code, some unnecessary code/tests are removed. So just to be clean, I apply formatting as well.